website/docs/reference/cli/index.md

```diff
@@ -0,0 +1,6 @@
+---
+title: CLI
+sidebar_label: Overview
+---
+
+The opctl cli provides command line access to using ops and managing a node.
```

Comment:
There is a broken link on https://opctl.io/docs/introduction, which doesn't have an appropriate page to link to. This is a quick and dirty fix (I don't want to introduce significant changes to the CLI docs in this PR).

Reply:
Good catch; this was probably my miss when I updated CLI docs to be hierarchical vs a single page. I wish [linking to categories was a thing in docusaurus](https://github.com/facebook/docusaurus/issues/2643); hopefully it gets in soon.